docs/projects/P2/1_teamprocess.md

```diff
@@ -106,33 +106,41 @@ During this meeting, discuss potential functional requirements of this project.
 
 Then, document these functional requirements in the form of user stories that follow the guidelines discussed in lecture. All of your user stories should be in the format of "As a [role], I want [function], so that [value]".
 
-As a team, you should also come up with a prioritization ranking for each user story (i.e. how essential is this user story to the overall functionality of the application to your stakeholders?) 
-
 **You should come up with at least two user stories per student in your group.**
 
 !!! note "Formulating User Stories"
-    Consider what are the different types of users that will be interacting with your system (your stakeholders) and what features they would want to have that the current NodeBB does not provide. You may want to reference features in existing Q&A systems that may be desirable, or conduct interviews with people who could be a potential user of this system.
+    Consider what are the different types of users that will be interacting with your system (your stakeholders) and what features they would want to have that the current NodeBB does not provide. You may want to reference features in existing Q&A systems (e.g. Piazza, Diderot) that may be desirable, or conduct interviews with your peers who could be a potential user of this system.
 
-Once you have your list of user stories, create a new GitHub Project board (click into Projects on your team repository). Then create two new columns to the left called **"User Stories"** and **"Backlog"**. At this point in time, you should have **"User Stories", "Backlog", "To-Do", "In Progress" and "Done"**, in that order from left to right. Feel free to add more columns if your teams decide that you need them during your development process
+As a team, you should then come up with a prioritization ranking for each user story. The prioritization should be based on two factors 
 
-Add your user stories into the **"User Stories"** column using the **“+ Add item”** button at the bottom, create draft issues for each of your user stories.
+- **Impact**: how essential is this user story to the overall functionality of the application to your stakeholders, how beneficial it would be to your stakeholders, and
+- **Effort**: how much estimated time would be needed to implement features to achieve each user story.
```

Comment:
nit: replace with "How much time/effort is required to implement this user story"